### PR TITLE
Lavamoat - protect all UI contexts

### PR DESCRIPTION
**Pull request by kumavis**
replaces https://github.com/MetaMask/metamask-extension/pull/12938
replaces https://github.com/MetaMask/metamask-extension/pull/12959

add lavamoat protections to the ui

### CODE REVIEW

**Review by brad-decker — APPROVED** [review]
LGTM

File: patches/zxcvbn+4.4.2.patch [hunk 1]
```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/zxcvbn/lib/matching.js b/node_modules/zxcvbn/lib/matching.js
```
Comment (digiwand) [inline patches/zxcvbn+4.4.2.patch:1]: Q: Do you think it would be helpful for us to add description comments at the top of the patch files?

**Review by digiwand — APPROVED** [review]
No blockers. Small question above

File: patches/zxcvbn+4.4.2.patch (hunk 1 above)
Comment (kumavis) [reply patches/zxcvbn+4.4.2.patch:1]: yeah thats a good idea, ill review our existing patches

**Pull request merged**